example/example/spiders/books_04_overrides_02.py

```diff
@@ -67,12 +67,12 @@ class BooksSpider(scrapy.Spider):
             ("toscrape.com", BTSBookListPage, BookListPage),
             ("toscrape.com", BTSBookPage, BookPage),
```

Comment:
Should it be replaced with ApplyRule calls as well?

Reply:
Although it seems 3-tuples are not deprecated in this PR, so that's probably out of scope.

Reply:
Yeah I was thinking about doing it in this other PR https://github.com/scrapinghub/scrapy-poet/pull/88 since this PR focuses on **web-poet**'s deprecation.